MSAL/test/app/MSALTestAppSettings.m

```diff
@@ -88,13 +109,19 @@ + (MSALTestAppSettings*)settings
     
     dispatch_once(&s_settingsOnce,^{
         s_settings = [MSALTestAppSettings new];
+        s_settings->_profile = [[s_profiles allValues] objectAtIndex:0];
```

Comment:
Does it remember the last chosen profile in NSUserDefaults (as it is in ADAL)? or it always returns the first profile? :)

Reply:
Talked to Rohit offline, it does remember the last choice :)